### DISCUSSION TITLE
Seq timestamps

### DISCUSSION THREAD

**Discussion by Santoshchv** [datalust/seq-tickets#1190] [Help]
Hello All,

Currently we are using Seq win event logger to gather our Windows events both System and Application. But whenever the seq is restarted it would send a ton of logs at once from the past as if they are logged at the time of restart. 

So is there a way to make Seq use client timestamps and only use server side timestamps when there is a predefined time difference between client and server timestamps

**Comment by nblumhardt** [comment]
Hi 👋 

No, currently there's nothing like this - it might be something to try changing on the client side. What library/app are you using to forward the events from the Windows Event Log?
